.github/workflows/linting.yml

```diff
@@ -24,7 +24,7 @@ jobs:
       - uses: actions/setup-python@v3
       - uses: isort/isort-action@master
         with:
-          requirementsFiles: "requirements.txt utils/cq/requirements.txt"
+          requirementsFiles: "requirements.txt"
```

Comment:
Shouldn't this just require `utils/cq/requirements.txt`?

Reply:
No.  If we use a python module then it needs to be installed to check the code that uses it.  The utils/cq file is where we pull in the linting tools so need everthing here - and one change with this PR is that requirments.txt now pulls in the other files.